src/cmd-list

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+
+import datetime
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+from cosalib.builds import Builds
+from cosalib.cmdlib import parse_date_string
+
+
+def main():
+    builds = Builds()
+    tags = {}
+    for tag in builds.get_tags():
```

Comment:
Before diving into the loops, this should probably check that there are any builds and exit with a message about that.

Reply:
Tweaked this!